Jenkinsfile

```diff
@@ -23,6 +23,8 @@ pipeline {
         DISABLE_DOWNLOAD_PROGRESS_OPTS = '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn '
         LINUX_MVN_RANDOM = '-Djava.security.egd=file:/dev/./urandom'
         COVERAGE_EXCLUSIONS = '**/test/**/*,**/itests/**/*,**/*Test*,**/sdk/**/*,**/*.js,**/node_modules/**/*,**/jaxb/**/*,**/wsdl/**/*,**/nces/sws/**/*,**/*.adoc,**/*.txt,**/*.xml,**/platform-solr-server-standalone/**/*'
+        OWASP_ANALYZER_OPTS = '-DarchiveAnalyzerEnabled=false -DopensslAnalyzerEnabled=false -DnspAnalyzerEnabled=false -DnuspecAnalyzerEnabled=false -DassemblyAnalyzerEnabled=false'
+        MUTE_LOGS = '-DitestLogLevel=FATAL -DsecurityLogLevel=FATAL'
```

Comment:
I'm fine with this change, but I could see it being somewhat controversial.

Reply:
I'd be more accepting of setting it to WARN or ERROR or something, but FATAL just feels like we're certainly going to be missing at least some context when FATAL errors do occur.